pkg/console/starter/starter.go

```diff
@@ -0,0 +1,108 @@
+package starter
+
+import (
+	"fmt"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/cache"
+	"time"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/console-operator/pkg/console/operator"
+	"github.com/openshift/console-operator/pkg/generated/clientset/versioned"
+	"github.com/openshift/console-operator/pkg/generated/informers/externalversions"
+)
+
+func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+
+	consoleOperatorClient, err := versioned.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+
+	const resync = 10 * time.Minute
+
+	// only watch a specific resource name
+	tweakListOptions := func(options *v1.ListOptions) {
+		options.FieldSelector = fields.OneTermEqualSelector("metadata.name", operator.ResourceName).String()
```

Comment:
you will need to figure out if you can actually only watch a single name since you had multiple object names

Reply:
Right, or if i need more than one `tweakListOptionsFor____`

Reply:
You would need to explicitly create different informers since there is no `OR` in field selectors.  It would likely be simpler to filter at the queue.